js/demo-app/src/index.js

```diff
@@ -183,7 +187,11 @@ function buildNavMarkup(navConfigs) {
     // Build Programmatic Button
     let currButton = document.createElement('button');
     currButton.setAttribute('class', `nav-id-${curr.id}`);
-    currButton.addEventListener('click', () => {
+    currButton.addEventListener('click', (e) => {
+      programmaticNav.querySelectorAll('button').forEach(el => {
```

Comment:
Should we move this code into `updateActiveNav` so it's in one place.  Then it can be called here on click and on routing events from the frame coordinator?

I left the fqRoute (poorly named) as a param to that method so that we could eventually support other types of active tracking